pbench/cli/agent/commands/metalog.py

```diff
@@ -0,0 +1,43 @@
+import click
+
+from pbench.agent.task.add_metalog_option import add_metalog_option
+
+#
+# pbench cli
+#
+@click.group()
+def metalog():
```

Comment:
Can you add a function description to `metalog()` describing why it is a "no-op" function?